packages/base/Dropdown/src/Dropdown/Dropdown.tsx

```diff
@@ -49,6 +54,7 @@ interface InternalProps
   /** Sets the desired behavior for an element's overflow */
   contentOverflow?: ContentOverflowType
   popperContainer?: HTMLElement
+  classes?: Classes & { popper?: string; content?: string }
```

Comment:
```suggestion
  classes?: { popper?: string; content?: string }
```

I believe we don't support any other class than these two